miscellaneous/metaDataRemover/txDataFormater.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"encoding/hex"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/builders"
+)
+
+const ESDTDeleteMetadataPrefix = "ESDTDeleteMetadata"
+
+func createShardTxsDataMap(shardTokensMap map[uint32]map[string]struct{}, tokensToDeletePerTx uint64) (map[uint32][][]byte, error) {
```

Comment:
why not a struct for this functionality?

Reply:
>Split txsCreator functionality in interfaces

As described in PR description, probably a refactor in that area will follow if necessary

Reply:
right, missed the description